cask-bootstrap.el

```diff
@@ -41,7 +41,7 @@
   "Path to Cask bootstrap directory.")
 
 (defconst cask-bootstrap-packages
-  '(s dash f commander git epl shut-up cl-lib package-build)
+  '(s dash f commander git epl shut-up cl-lib package-build eieio)
```

Comment:
I don't think built-in packages need adding here?

Reply:
Shouldn't it be required nevertheless?
My memory is a bit weak now but I think it was only emacs 24.3 that failed without this. Unfortunately I can't run this now locally for 24.3. I can do a fixup and see what travis says about it if you like?

Reply:
I thought that was due to melpa/package-build#19? I don't think eieio is available from any package repository anyway, and isn't used directly by cask?